pagopa/io-app
# chore: Push notification grouping

## Short description
This PR groups the push-notification's related files into `features/pushNotifications`.

## List of changes proposed in this pull request
- All files directly related to the handling of push notifications have been moved under `features/pushNotifications`
- Removed `default` exports in favour or direct ones
- Profile's push notification screens have been left under the profile folders, since they are attributes of the profile and the user preferences

## How to test
Run the application on the physical device and check that everything push-notification-related is working accordingly.
